src/components/Timer/Timer.tsx

```diff
@@ -10,46 +10,49 @@ interface Timer {
 
 const TimeRemaining: React.FC = () => {
   const calculateTimeLeft = () => {
-    const difference = +new Date(`1-14-2022`) - +new Date()
-    let timeLeft = {
-      days: "0",
-      hours: "0",
-      minutes: "0",
-      seconds: "0",
-    }
+    // Fri Jan 14 2022 15:00:00 GMT-0800 (Pacific Standard Time)
+    const eventDate = new Date("2022-01-14T15:00-08:00")
 
-    if (difference > 0) {
-      const days = Math.floor(difference / (1000 * 60 * 60 * 24))
-      const hours = Math.floor((difference / (1000 * 60 * 60)) % 24)
-      const minutes = Math.floor((difference / 1000 / 60) % 60)
-      const seconds = Math.floor((difference / 1000) % 60)
-      timeLeft = {
-        days: (
-          (days < 100 ? "0" : "") +
-          (days < 10 ? "0" : "") +
-          days
-        ).toString(),
-        hours: ((hours < 10 ? "0" : "") + hours).toString(),
-        minutes: ((minutes < 10 ? "0" : "") + minutes).toString(),
-        seconds: ((seconds < 10 ? "0" : "") + seconds).toString(),
-      }
+    const difference = +eventDate - Date.now()
+
+    if (difference < 0) {
+      return undefined
     }
 
-    return timeLeft
+    const days = Math.floor(difference / (1000 * 60 * 60 * 24))
+    const hours = Math.floor((difference / (1000 * 60 * 60)) % 24)
+    const minutes = Math.floor((difference / 1000 / 60) % 60)
+    const seconds = Math.floor((difference / 1000) % 60)
+
+    return {
+      // (days < 100 ? "0" : "") +
+      days: ((days < 10 ? "0" : "") + days).toString(),
+      hours: ((hours < 10 ? "0" : "") + hours).toString(),
+      minutes: ((minutes < 10 ? "0" : "") + minutes).toString(),
+      seconds: ((seconds < 10 ? "0" : "") + seconds).toString(),
+    }
   }
 
   const [timeLeft, setTimeLeft] = useState<Timer>({
-    days: "0",
-    hours: "0",
-    minutes: "0",
-    seconds: "0",
+    days: "00",
+    hours: "00",
+    minutes: "00",
+    seconds: "00",
   })
 
   useEffect(() => {
-    setTimeout(() => {
-      setTimeLeft(calculateTimeLeft())
-    }, 1000)
-  })
+    const timer = setInterval(() => {
+      const val = calculateTimeLeft()
+      if (!val) {
+        clearInterval(timer)
+        return
+      }
+
+      setTimeLeft(val)
+    }, 500)
```

Comment:
I also changed this to half a second to alleviate slight delays when I try to sync the timer between multiple windows.